src/assertions/AssertionGenerator.js

```diff
@@ -1,6 +1,8 @@
 import UnexpectedHtmlLike from 'unexpected-htmllike';
 import React from 'react';
+import { eventTypes } from 'react-dom/lib/SimpleEventPlugin';
```

Comment:
I wonder about just hard coding the list for this - this might move at any time, but the list of supported events shouldn't change (and adding/removing one is much easier and lower impact than if this just breaks because `react-dom/lib/SimpleEventPlugin` changes its name :) )

wdyt?